Tools/TestWebKitAPI/Configurations/TestWebKitAPI-macOS.entitlements

```diff
@@ -2,15 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>keychain-access-groups</key>
-	<array>
-		<string>com.apple.TestWebKitAPIAlternate</string>
-		<string>com.apple.TestWebKitAPI</string>
-	</array>
-	<key>com.apple.private.webkit.webpush</key>
-	<true/>
-	<key>com.apple.private.webkit.webpush.inject</key>
-	<true/>
```

Comment:
The commit message says that we can remove entitlements from open source build, but this seems to remove them altogether. I understand the WKTR change, but not the TestWebKitAPI one.

Reply:
There is a separate, pre-existing `Tools/TestWebKitAPI/Configurations/TestWebKitAPI-macOS-internal.entitlements` which provides internal-only entitlements. The changes to WKTR make it behave more like TestWebKitAPI in this area.